### PR TITLE
minor: replace original apex-core repository with checkstyle mirror f…

### PR DESCRIPTION
**Pull request by nmancus1**
minor: replace original apex-core repository with checkstyle mirror for regression testing

### CODE REVIEW

File: .ci/wercker.sh [hunk 1]
```diff
@@ -100,11 +100,11 @@ no-error-apex-core)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  checkout_from https://github.com/apache/incubator-apex-core.git
-  cd .ci-temp/incubator-apex-core
+  checkout_from https://github.com/apache/apex-core
```
Comment (rnveach) [inline .ci/wercker.sh]: @nmancus1 I believe checkstyle forks are all part of the checkstyle organization. Ex: https://github.com/checkstyle/apex-core

**Comment by rnveach** [comment]
https://github.com/checkstyle/checkstyle/wiki/PR-rules
Only commit in PR unless we specify otherwise.

File: .ci/wercker.sh (hunk 1 above)
Comment (nmancus1) [reply .ci/wercker.sh]: Yes, that's right.  I had the old version forked as well, and it caused me some confusion. Fixed.

**Comment by nmancus1** [comment]
> https://github.com/checkstyle/checkstyle/wiki/PR-rules
> Only commit in PR unless we specify otherwise.

I don't understand; did I do otherwise?

**Comment by rnveach** [comment]
You did right. Forgot the `one` in `Only one commit`.

**Pull request merged**